index.html

```diff
@@ -17,8 +17,8 @@
           </div>
 
           <ul class="nav-links">
-            <li><a href="#">About</a></li>
-            <li><a href="#">Services</a></li>
+            <li><a href="#">Sobre nosotros</a></li>
+            <li><a href="#">Servicios de barbería</a></li>
```

Comment:
Por qué no lo cambias por
```suggestion
            <li><a href="#">Servicios de Barbería</a></li>
```